_translations/it.yml

```diff
@@ -180,6 +180,7 @@ it:
     walletelectrum: "Electrum punta a velocità e semplicità, con un basso impiego di risorse. Utilizza dei server remoti che gestiscono le parti più complicate del sistema Bitcoin, e permette il recupero del proprio portafoglio con una frase segreta."
     walletmsigna: "mSIGNA è un portafoglio avanzato e di facile utilizzo che fornisce velocità e semplicità, scalabilità di livello aziendale e una robusta sicurezza. Supporta BIP32, transazioni multi-firma, archiviazione offline, sincronizzazione su più dispositivi, copie di backup elettroniche cifrate e anche cartacee."
     walletbitcoinwallet: "Bitcoin Wallet è facile da usare e affidabile, pur essendo anche sicuro e veloce. La sua visione è la decentralizzazione e nessun bisogno di fiducia; nessun servizio centrale è necessario per le operazioni riguardanti il Bitcoin. L'app è una buona scelta per gli utenti non tecnici."
+    walletedgewallet: "Edge è un portafoglio mobile di Bitcoin, compatibile con Segwit che rende gli alti livelli di privacy, sicurezza e decentralizzazione molto familiari e utilizzabili da chiunque. I portafogli Edge sono sempre crittografati e sottoposti a backup in modo automatico e funzionano anche quando i server Edge non funzionano."
```

Comment:
Please revise or drop change to this file as per comment in: `_translations/en.yml`